deployments/ansible/roles/collector/templates/splunk-otel-collector.conf.j2

```diff
@@ -19,8 +19,8 @@ SPLUNK_HEC_URL={{ "https://ingest." + splunk_realm + ".signalfx.com/v1/log" }}
 {% endif %}
 SPLUNK_HEC_TOKEN={{ splunk_hec_token }}
 SPLUNK_MEMORY_TOTAL_MIB={{ splunk_memory_total_mib }}
-SPLUNK_BALLAST_SIZE_MIB={{ splunk_ballast_size_mib }}
 {% if splunk_listen_interface is defined and splunk_listen_interface %}
+GOMEMLIMIT={{ go_mem_limit }}
 SPLUNK_LISTEN_INTERFACE={{ splunk_listen_interface }}
 {% endif %}
```

Comment:
Move `go_mem_limit` to a separate `if` block:
```suggestion
{% if splunk_listen_interface is defined and splunk_listen_interface %}
SPLUNK_LISTEN_INTERFACE={{ splunk_listen_interface }}
{% endif %}
{% if go_mem_limit is defined and go_mem_limit %}
GOMEMLIMIT={{ go_mem_limit }}
{% endif %}
```

Reply:
done